VmAgent.Core/Interfaces/DockerContainerEngine.cs

```diff
@@ -358,7 +359,13 @@ public async Task<SessionHostInfo> CreateAndStart(int instanceNumber, GameResour
             // is the same as the remote image path.
             SessionHostsStartInfo sessionHostStartInfo = gameResourceDetails.SessionHostsStartInfo;
             ContainerImageDetails imageDetails = sessionHostStartInfo.ImageDetails;
-            string imageName = $"{imageDetails.Registry}/{imageDetails.ImageName}:{imageDetails.ImageTag ?? "latest"}";
+            string imageName = $"{imageDetails.ImageName}:{imageDetails.ImageTag ?? "latest"}";
```

Comment:
thanks for the PR! Can you wrap the change in a `if sessionHostManager.LinuxContainersOnWindows` clause, to make sure this applies only on LocalMultiplayerAgent and not on VmAgent running on production VMs? I don't think this will have a negative impact there, but I'd rather not change existing behavior unless it's absolutely necessary. I'll also make a note on the contribution notes after the PR that changes in the .Core project that target LocalMultiplayerAgent should make use of this flag. Thanks again!

Reply:
Hi @dgkanatsios , I don't mind adding that at all but is it the right guard? Was just luck that I didn't run into this issue when testing window containers due to the image names.

Reply:
got it, you're right. We're good, thanks!